app/src/features/datasets/components/security/ApplySecurityDialog.tsx

```diff
@@ -13,43 +14,106 @@ import YesNoDialog from 'components/dialog/YesNoDialog';
 import { DialogContext, ISnackbarProps } from 'contexts/dialogContext';
 import { Formik, FormikProps } from 'formik';
 import { useApi } from 'hooks/useApi';
-import { IArtifact } from 'interfaces/useDatasetApi.interface';
+import useDataLoader from 'hooks/useDataLoader';
+import { IArtifact, IPersecutionAndHarmRule } from 'interfaces/useDatasetApi.interface';
 import React, { useContext, useRef, useState } from 'react';
+import { pluralize as p } from 'utils/Utils';
```

Comment:
great idea, for utils function! Is there any reasons for casting as `p`? Only a nit, but sometimes nice to see function name to understand intention without scrolling up. Either way nice work!

Reply:
Cheers! I shortened it to `p` as to increase the readibility of the template strings; Throwing in `pluralize(1, 'word')` breaks up the sentence a bit, compared to `p(1, 'word')`